roles/kubernetes/node-label/tasks/main.yml

```diff
@@ -35,15 +35,29 @@
     - node_labels is defined
     - node_labels is mapping
 
+- name: Set multus role node label to empty list
```

Comment:
Why not have the deployer do this in their inventory instead of adding arbitrary labels in kubespray code? The purpose for the `node-label` role is to set node labels based on inventory variables not compute them based on arbitrary input.

Reply:
Sorry, I didn’t get your proposal with deployer. Can you explain more details what you mean exactly ? The problem exists exactly with mixed type of containers engine inside one cluster. So, in this situation we have to schedule pods per group of nodes with specific container engine and the configuration of each daemonset is deferent. There is no universal configuration for all container engines. Can we do that not only with `nodeSelector` ? I couldn’t another way and any other standard labels which are existing without adding something new.

Reply:
deployer = human operator, when you compose your ansible inventory you can add node labels in the inventory and kubespray will add them to the nodes. There is no need for kubespray to add "standard" labels as there is no _standard_ to speak of. `nodeSelector` is the standard way of targeting workloads on specific nodes.

Reply:
Well, I understood your point of view and will re-work. I tried to prepare to get this as working automatically without involving and making some preparation. So, we will need to put some info how to manage this for each deployer with mixed container type cluster. I guess that multus doc will be a good place for this, right ?